tests/test_process.py

```diff
@@ -90,6 +91,222 @@ def test_upload_dataversion_metadata_without_team_images_id(self, syn: Any):
         )
 
 
+class TestApplyCustomTransformations:
+    """Test the apply_custom_transformations function."""
+
+    @pytest.fixture(scope="function", autouse=True)
+    def standard_transform_function(
+        self,
+    ) -> Callable[[pd.DataFrame, str, Dict[str, pd.DataFrame]], pd.DataFrame]:
+        """mock simple transform function that uses standard parameters such as dataframe, dataset_name, datasets"""
+
+        def _standard_transform_function(
+            df: pd.DataFrame, dataset_name: str, datasets: Dict[str, pd.DataFrame]
+        ) -> pd.DataFrame:
+            """mock simple transform function"""
+            datasets["new_key"] = 2
```

Comment:
I don't understand what this line is doing if we're overwriting the value in the return?